cli/src/orderbook.rs

```diff
@@ -0,0 +1,30 @@
+use asset_identifier;
+use clap::ArgMatches;
+use error::Result;
+use fmt::{Formatter, Simple};
+use stellar_client::{endpoint::{orderbook, Limit}, sync::Client};
+
+pub fn details(client: &Client, matches: &ArgMatches) -> Result<()> {
+    let endpoint = {
+        let base_asset_str = matches
+            .value_of("base_asset")
+            .expect("Base asset is a required field");
+        let counter_asset_str = matches
+            .value_of("counter_asset")
+            .expect("Counter asset is a required field");
+        let base_asset = asset_identifier::parse_identifier_from_str(base_asset_str)?;
```

Comment:
this is really long: `asset_identifier::parse_identifier_from_str`

I think we should consider that if we are using this from the module, the name of the method can probably just be `from_str`:

```rust
let base = asset_identifier::from_str(base_str)?;
```